src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1

```diff
@@ -105,9 +105,12 @@ class AutoBugLog : EventBase  {
                 else {
                     $resourceOwner = $AssignedTo;
                 }
+
+                #check for user is active or not
+                $isUserActive = $metaProviderObj.isUserActive($this.OrganizationName, $AssignedTo);
```

Comment:
Should it be $resourceOwner instead of $AssignedTo?

Reply:
no resource owner is just to display in bug description, we log bug for assignee